zk/stages/stage_sequence_execute_utils.go

```diff
@@ -48,7 +48,7 @@ const (
 
 	transactionGasLimit = 30000000
 
-	yieldSize = 100 // arbitrary number defining how many transactions to yield from the pool at once
+	yieldSize = 1000 // arbitrary number defining how many transactions to yield from the pool at once
```

Comment:
let's do it a cli flag, with the default value of 1000

Reply:
done